pom.xml

```diff
@@ -35,7 +35,7 @@
   <properties>
     <version.eclipse.microprofile.jwt>1.1.1</version.eclipse.microprofile.jwt>
     <version.microprofile.config>1.3</version.microprofile.config>
-
+    <version.smallrye.config>1.4.1</version.smallrye.config> 
```

Comment:
there's actually a newer 1.5.0 out, but the groupId changed to io.smallrye.config